src/main/jbake/content/documentation/bundles/org-apache-sling-junit-bundles.md

```diff
@@ -6,7 +6,7 @@ tags=testing
 
 This is an overview of the Sling bundles that provide support for server-side JUnit tests. 
 
-The Maven modules below [`testing/samples`](https://svn.apache.org/repos/asf/sling/trunk/testing/samples) 
+The Maven modules below [`samples/testing`](https://svn.apache.org/viewvc/sling/trunk/samples/testing)
```

Comment:
Sorry, I wasn't specific enough in Jira issue. All code is under github, including the samples at https://github.com/apache/sling-samples

Reply:
@rombert, please, check if it ok now.